tests/project-type/workspace-instance.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { WorkspaceImpl } from "@sap/project-api";
+import { expect } from "chai";
+import { uniq } from "lodash";
+
+import { mockVscode } from "../mockUtil";
+
+const testVscode = {
+    workspace: {
+        workspaceFolders: () => []
+    }
+};
+
+mockVscode(testVscode, "src/project-type/workspace-instance.ts");
```

Comment:
Can we avoid mocking VSCode here by using DI ? (Pure unit tests)

Reply:
I did not find. I added mock objects to other tests

Reply:
needing new mocks for existing mocks based tests makes sense.
but why does this test needs mocks?